pyk/src/pyk/k2lean4/k2lean4.py

```diff
@@ -154,7 +154,7 @@ def _collection(self, sort: str) -> Structure:
                 val = Term(f'List {item}')
             case CollectionKind.MAP:
                 key, value = sorts
-                val = Term(f'List ({key} × {value})')
+                val = Term(f'(MapHook {key} {value}).map')
```

Comment:
Does the program still compile? How much time does it take?

Reply:
Yes, it does compile. The user needs to add the prelude, add `open MapHookDef` and `set_option maxHeartbeats 10000000`

Reply:
Maybe with fewer heartbeats, it also compiles.  I find it takes around the same time as before, 2-3 minutes.